Mathlib/LinearAlgebra/QuadraticForm/Dual.lean

```diff
@@ -52,14 +51,15 @@ section Ring
 variable [CommRing R] [AddCommGroup M] [Module R M]
 
 theorem nondenerate_dualProd :
-    (dualProd R M).Nondegenerate ↔ Function.Injective (Module.Dual.eval R M) := by
+    (LinearMap.toBilin (dualProd R M)).Nondegenerate
+    ↔ Function.Injective (Module.Dual.eval R M) := by
```

Comment:
There is [LinearMap.SeparatingLeft](https://leanprover-community.github.io/mathlib4_docs/Mathlib/LinearAlgebra/SesquilinearForm.html#LinearMap.SeparatingLeft) which is the same as `BilinForm.Nondegenerate`.

Reply:
That's better, thanks.